Source/Models/PHSettings.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  PHSettings.swift
+//  Product Hunt
+//
+//  Created by Vlado on 4/27/16.
+//  Copyright © 2016 ProductHunt. All rights reserved.
+//
+
+import Foundation
+
+struct PHSettings {
```

Comment:
For me having Actions, Reducer and state structure in one `module` file can be hard to follow